### PR TITLE
feat: add Polygon, Base and Base Sepolia in static chain info

### PR DESCRIPTION
**Pull request by lumtis**
# Description

As part of v18

[upgrade tests pass](https://github.com/zeta-chain/node/actions/runs/9910599831)

### CODE REVIEW

**Review by ws4charlie — APPROVED** [review]

**Review by kingpinXD — APPROVED** [review]

File: docs/openapi/openapi.swagger.yaml [hunk 1]
```diff
@@ -53612,6 +53612,10 @@ definitions:
       - goerli_localnet
       - btc_regtest
       - amoy_testnet
+      - optimism_mainnet
```
Comment (swift1337) [inline docs/openapi/openapi.swagger.yaml:53615]: Are OP stack keys placed on purpose?

File: app/setup_handlers.go [hunk 2]
```diff
@@ -5,12 +5,9 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
-	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
-	lightclienttypes "github.com/zeta-chain/zetacore/x/lightclient/types"
-	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
 )
 
-const releaseVersion = "v17"
```
Comment (gartnera) [inline app/setup_handlers.go:13]: Do we really need to do a full major version for this change? This seems more like a v17.1.0 but requires an upgrade proposal.

File: docs/openapi/openapi.swagger.yaml (hunk 1 above)
Comment (lumtis) [reply docs/openapi/openapi.swagger.yaml:53615]: To keep consistency with the field numbers on `develop` although these values are not used here

File: app/setup_handlers.go (hunk 2 above)
Comment (lumtis) [reply app/setup_handlers.go:13]: This is a consensus breaking changes and we decided to do a major release for every consensus breaking changes, unless we discuss changing this recently?

Comment (gartnera) [reply app/setup_handlers.go:13]: Ok

**Review by gartnera — APPROVED** [review]

**Comment by lumtis** [comment]
Upgrade test successful completion: https://github.com/zeta-chain/node/actions/runs/9910599831/job/27381349189

File: pkg/chains/chain.go [hunk 3]
```diff
@@ -91,7 +91,10 @@ func IsZetaChain(chainID int64) bool {
 func IsEVMChain(chainID int64) bool {
 	return chainID == 5 || // Goerli
 		chainID == AmoyChain().ChainId ||
-		chainID == SepoliaChain().ChainId || // Sepolia
+		chainID == PolygonChain().ChainId ||
+		chainID == SepoliaChain().ChainId ||
+		chainID == BaseMainnetChain().ChainId ||
```
Comment (brewmaster012) [inline pkg/chains/chain.go:96]: should we add optimism chains here

**Review by brewmaster012 — APPROVED** [review]
LGTM

**Review by CharlieMc0 — APPROVED** [review]

**Pull request merged**